index.js

```diff
@@ -1,12 +1,84 @@
-const express = require('express');
-const helmet = require('helmet');
-
-const server = express();
-
+const express    = require('express');
+const helmet     = require('helmet');
+const knex       = require('knex');
+const knexConfig = require('./knexfile.js');
+const server     = express();
 server.use(express.json());
 server.use(helmet());
-
+const db         = knex(knexConfig.development);
 // endpoints here
+server.get('/', (req, res) => {
```

Comment:
I would encourage you here to continue to use router even if it isn't required. It's great practice and will keep you sharp for build week.